src/pages/cart/Cart.jsx

```diff
@@ -0,0 +1,13 @@
+import { Footer, Navbar } from "../../components";
+import "./Cart.css"
+import { Main } from "./main/Main";
```

Comment:
try using relative path instead of absolute path

Reply:
used absolute path instead of relative path